gp-templates/translations.php

```diff
@@ -105,6 +105,13 @@
 		$filters_values_only = array_filter( $filters );
 		$sort_values_only    = array_filter( $sort );
 		$filters_and_sort    = array_merge( $filters_values_only, $sort_values_only );
+		// Remove any non-string or non-numeric values from the array.
+		$filters_and_sort    = array_filter(
+			$filters_and_sort,
+			function ( $value ) {
+				return is_string( $value ) || is_numeric( $value );
+			}
+		);
```

Comment:
Similar to above, this can be simplified to scalar with:

```suggestion
		$filters_and_sort    = array_filter( $filters_and_sort, 'is_scalar' );
```

Reply:
@dd32 Thank you for the feedback! I have committed your suggestions. Can you re-approve the PR another time? Thank you!